orchagent/tunneldecaporch.cpp

```diff
@@ -40,19 +40,36 @@ void TunnelDecapOrch::doTask(Consumer& consumer)
         string key = kfvKey(t);
         string op = kfvOp(t);
 
-        IpAddresses ip_addresses;
-        IpAddress src_ip;
+        IpAddresses dst_ip_addresses;
+        IpAddress src_ip("0.0.0.0");
```

Comment:
Where are we getting the src_ip? If its from the DB data, then P2P can be determined based on the presence of src_ip and not the MUX tunnel name

Reply:
The src_ip is from config db. I was also thinking to determine the term type based on if there is `src_ip` or not. But I'm afraid we will also have `src_ip` for P2MP in the future. That's why I used the hardcoded mux tunnel name to make the decision.

Reply:
I think we can design to not have src_ip with P2MP.

Reply:
Sure. I will update the logic here if that is the case. Thanks